Makefile

```diff
@@ -63,7 +63,7 @@ misc/certs/ca-certificates.crt:
 	docker run "amazon/amazon-ecs-agent-cert-source:make" cat /etc/ssl/certs/ca-certificates.crt > misc/certs/ca-certificates.crt
 
 test:
-	. ./scripts/shared_env && go test -timeout=25s -v -cover $(shell go list ./agent/... | grep -v /vendor/)
+	. ./scripts/shared_env && go test -race -timeout=25s -v -cover $(shell go list ./agent/... | grep -v /vendor/)
```

Comment:
Can you update [this line](https://github.com/aws/amazon-ecs-agent/blob/v1.14.3/appveyor.yml#L16) too so we get race detection on Windows as well?